acceptance-tests/build.gradle

```diff
@@ -35,5 +36,7 @@ task acceptanceTest(type: Test) {
   mustRunAfter rootProject.subprojects*.test
   description = 'Runs EthSigner acceptance tests.'
   group = 'verification'
+
+  useJUnitPlatform()
```

Comment:
What does this do? ...is it a built in Gradle op? ...if so, which version of Gradle i.e. does it work with 4.x or 5.x?

Reply:
It's built into gradle as part of the Java plugin. Included in gradle 4.6+. We are using gradle 5.2.1 in EthSigner.

https://docs.gradle.org/4.6/release-notes.html

Reply:
Good to know 👍